optimization/optimization.md

```diff
@@ -72,10 +74,11 @@ Look up the advanced optimization concepts introduced in the slides:
 3. Precaching
 
 4. Concurrency
-- Different languages have different concurrency models? How does the Javascript one work?
-- What is a stack, heap and queue?
+- Do different languages have different concurrency models? How does Javascript one work?
+- What are stack, heap, and queue?
 - What is the Javascript event loop?
 
-### More Resources
-- [Video: Minifying JS and CSS](https://www.youtube.com/watch?v=8aAuNQqgGAg)
-- [Video: Caching Techniques](https://www.youtube.com/watch?v=RgPf5RDv4-s)
+### Supplemental Materials
+- [Minifying JS and CSS video (6 mins watch)](https://www.youtube.com/watch?v=8aAuNQqgGAg)
```

Comment:
Can you bring these three back to materials and ask learners to go over them in the ### lesson?